### ISSUE TITLE
Undefined variable: this

### ISSUE THREAD

**Issue by designcise** [HawkBitPhp/hawkbit#34]
**Issue:** Inside an application closure function, ```$this``` is undefined.

**To Reproduce:**
```
/// ...
$app->get('/', function (Request $request, Response $response) {
    echo nl2br(print_r($this, true)); // or try $temp = $this;
    return $response;
});
```

**Expected Result:** Should return an instance of the Hawkbit\Application object.

The same thing happens inside ```$app->group()```

---

**Note:** Also on an unrelated note, this link in the docs leads to a 404 "[http://container.thephpleague.com/service-providers/](http://container.thephpleague.com/service-providers/)"

**Comment by mbunge** [comment]
Hey, thanks. 

Interessting error. Hawkbit binds it self automatically to closure.

A workaround until fix is to declare a use on $app.

**Comment by designcise** [comment]
@mbunge yes, already using ```use``` would be great to have the instance bound to closure though.

**Comment by memran** [comment]
i m facing same.

**Comment by mbunge** [comment]
Fixed with release 2.3.2 🎉

**Comment by mbunge** [comment]
Fixed with release 2.3.2 🎉
